libvirt/README.md

```diff
@@ -58,3 +58,224 @@ mkdir -p /opt/data-dir
 
 ```
 
+# Creating an end-to-end environment for testing and development
+
+In this section you will learn how to setup an environment in your local machine to run peer pods with
+the libvirt cloud API adaptor. Bear in mind that many different tools can be used to setup the environment
```

Comment:
I think it's better to mention that this requires a Linux/KVM system with libvirt. Some devs do use Mac ;)